db/changelog/v-1/03-changeset-test-status-table.xml

```diff
@@ -5,26 +5,20 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <changeSet id="10" author="aktsay" context="dev or prod">
+    <changeSet id="10" author="kentr0w" context="dev or prod">
         <preConditions onFail="MARK_RAN">
             <not>
-                <tableExists tableName="test_status"/>
+                <tableExists tableName="test_execution"/>
```

Comment:
This file should be renamed from status to executino too

Reply:
Renamed